pkg/utils/miscellaneous.go

```diff
@@ -211,20 +219,20 @@ func StorageProviderFromInfraProvider(infra *druidv1alpha1.StorageProvider) (str
 	}
 
 	switch *infra {
-	case aws, s3:
```

Comment:
Could not leave the comment at an appropriate place, so leaving it here. Function `ValueExists` is not used anywhere. Do you need it?